cmd/mapstructure-to-hcl2/mapstructure-to-hcl2.go

```diff
@@ -415,7 +415,11 @@ func getMapstructureSquashedStruct(topPkg *types.Package, utStruct *types.Struct
 		if _, ok := field.Type().(*types.Signature); ok {
 			continue // ignore funcs
 		}
-		structtag, _ := structtag.Parse(tag)
+		structtag, err := structtag.Parse(tag)
+		if err != nil {
+			log.Printf("could not parse field tag %s of : %v", tag, err)
+			continue
+		}
```

Comment:
This could also be a cause to the panic in #8511 (https://circleci.com/gh/hashicorp/packer/27449?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)